lib/gosu/patches.rb

```diff
@@ -45,6 +45,11 @@ def eql?(other)
   end
 end
 
+module Gosu
+  # Backwards compatibility: Channel was called SampleInstance before Gosu 0.13.0.
+  SampleInstance = Channel
```

Comment:
Just a minor nitpick - wouldn't it be better to move it to compat and use deprecate_const like we did it with the GOSU_COPYRIGHT_NOTICE?  And even if you don't like the deprecation warning it would be more consistent in the compat file because 

> # This whole file is about backward compatibility.

Otherwise nice job :smile:

Reply:
Oh wow, you are right. Thanks for pointing that out! Will fix it ASAP.